### ISSUE TITLE
groupTitlesWidget not actually displaying for ListCustomGroupedCheckbox

### ISSUE THREAD

**Issue by twostripe** [liodali/checkbox_grouped#65]
When using a `groupTitlesWidget` in a `ListCustomGroupedCheckbox` no actual `groupTitlesWidget` appears.

Looking at the code its clear why:

```
groupTitle:
    (widget.groupTitles != null && widget.groupTitles!.isNotEmpty)
        ? Container(
            alignment: widget.titleGroupedAlignment,
            padding: widget.titlePadding,
            child: Text(
              widget.groupTitles![index],
              style: widget.titleGroupedTextStyle ??
                  Theme.of(context).textTheme.headline6,
            ),
          )
        : null,
```

^ it's checking if the `widget.groupTitles` is null, but not using the `widget.groupTitlesWidget` instead.

Here's my proposed solution:
```
groupTitle:
    (widget.groupTitles != null && widget.groupTitles!.isNotEmpty)
        ? Container(
            alignment: widget.titleGroupedAlignment,
            padding: widget.titlePadding,
            child: Text(
              widget.groupTitles![index],
              style: widget.titleGroupedTextStyle ??
                  Theme.of(context).textTheme.headline6,
            ),
          )
        : (widget.groupTitlesWidget != null && widget.groupTitlesWidget!.isNotEmpty)
          ? widget.groupTitlesWidget![index]
          : null,
```

**Comment by liodali** [comment]
yeah i forget it thanks man,
sorry for late i will publish quick fix

**Comment by liodali** [comment]
i published new version `1.8.1`
